apps/gateway/src/routes/app.ts

```diff
@@ -17,7 +17,13 @@ import telemetry from '$/routes/v3/otlp'
 const app = createApp()
 
 if (process.env.NODE_ENV !== 'test') {
-  app.use(logger())
+  app.use('*', async (c, next) => {
+    if (c.req.path !== '/health') {
```

Comment:
Also exclude `api-docs` and `docs` which open api documentation

https://gateway.latitude.so/api-docs/
https://gateway.latitude.so/doc